SI/Sem.05/Pract.05/README.md

```diff
@@ -1 +1,71 @@
-TBA
\ No newline at end of file
+## Задачи
+
+1. Напишете програма, която намира първия повтарящ се елемент в редица от N числа. <br />
+
+2. Напишете програма, която подрежда елементите на редица от N числа, като първите елементи са четните, вторите - нечетните. <br />
+   Бонус: Сортирайте четните във възходящ ред,а нечетните в низходящ ред(отново четните са първи,а нечетните - втори). <br />
+
+3. Напишете програма, която превръща дадено естествено число в двоична бройна система. <br />
+
+4. Напишете програма, която извежда дали едно число е палиндром. <br />
+
+5. Напишете функция, която по два въведени масива принтира в конзолата тяхното сечение. <br />
+   **Бонус:** Напишете функция, която принтира в конзолата сечението на два масива, които са предварително сортирани. Функцията трябва да прави най-много по едно обхождане на всеки от масивите!
+
+6. Напишете програма, която подрежда елементите на редица от N числа по следния начин: <br />
+    а1 > а2 < а3 > а4 <(...) <br />
+
+7. Напишете функция, която приема матрица NxN и транспонира матрицата.
+
+8. Напишете функция, която приема матрица NxM и матрица MxK и умножава първата матрица по втората.
+
+## Tic-Tac-Toe
+
+UI         |  CLI
+:-------------------------:|:-------------------------:
+![tic-tac-toe-UI](https://images.ctfassets.net/3prze68gbwl1/asset-17suaysk1qa1jp4/fe212fc1d9d90b12907f5b6b04fa9280/android-ios-tic-tac-toe.png?fm=avif&w=500&h=600&fit=) | ![tic-tac-toe-CLI](https://user-images.githubusercontent.com/44736064/62582421-bd23a580-b882-11e9-92d8-ba0dced4bfb4.png)
+
+### [Онлайн версия на играта](https://playtictactoe.org)
+<br />
+
+Да се реализира в **конзолна** играта  **Тic-Tac-Toe**(морски шах). <br />
+Играта започва с празна дъска, като на всяка стъпка:
+- единият играч **въвежда две числа** - **ред и колона**, които са неговия ход на дъската(Например: **1 3**).
+- след всеки ход се актуализира конкретното състояние на играта.
+
+Накрая се показва кой е спечелил играта.
+
+
+
+
+## Мinesweeper
```

Comment:
This will be an overkill for the students

Reply:
Could leave it for whoever wants to try it & has the time?

Or are we strictly against it ?